docs/developer-guide/nodejs-api.md

```diff
@@ -242,6 +241,34 @@ The return value is an object containing the results of the linting operation. H
 }
 ```
 
+If the operation ends with a parsing error, you will get a single message for this file, with `fatal: true` added as an extra property.
+
+```js
+{
+    results: [
+        {
+            filePath: "./myfile.js",
+            messages: [
+                {
+                    ruleId: null,
+                    fatal: true,
+                    severity: 2,
+                    source: "fucntion foo() {}",
+                    message: "Parsing error: Unexpected token foo",
```

Comment:
This is fine. `fucntion` is treated as a valid identifier, so the unexpected token is actually `foo`, since you can't have two identifiers separate by a space in this context.